.changeset/brown-needles-hope.md

```diff
@@ -0,0 +1,5 @@
+---
+'@kadena/react-ui': patch
+---
+
+Used new token system
```

Comment:
It would be good if these comments are more descriptive. What components were updated for example